packages/loot-core/src/platform/server/indexeddb/index.ts

```diff
@@ -0,0 +1,113 @@
+let openedDb: null | ReturnType<typeof _openDatabase> = _openDatabase();
```

Comment:
This file is pretty much the same as `index.web.ts` - except it is now strict TS and the `.d.ts` type definitions have been inlined.

See the file below with comments about the potentially functional changes I did in this file.